pkg/crc/network/utils.go

```diff
@@ -92,3 +94,24 @@ func UriStringForDisplay(uri string) (string, error) {
 	}
 	return uri, nil
 }
+
+func CheckCRCLocalDNSReachableFromHost(clusterName, domainName, appsDomainName string) error {
+	ip, err := net.LookupIP(fmt.Sprintf("api.%s.%s", clusterName, domainName))
```

Comment:
@cfergeau Did we test this on OSx since api entry is only added to hosts file so not sure if `net.LookupIP` works for it? we need to test the osx build which is cross compiled on linux, build which is done on osx will not going error out for same.

Reply:
Yes, I tested it on macos yesterday, and did not have issues. However, this was a binary built on the macos machine, not a cross-build, now that you raised this question, this deserves additional testing (but I don't have access to the mac today).

Reply:
it should, but please verify a cross-build. do macOS based builds use CGO_ENABLED?

Reply:
> do macOS based builds use CGO_ENABLED?

No, that way we are not able to make cross builds :(

I tested the cross bit from centos artifact and it is working.

Reply:
Should we also verify it on windows before merging it?

Reply:
Would not hurt if someone has a Windows install handy for testing.

Reply:
will perfrom a test, but currently have limited access to resources.

Reply:
I did tested this on windows :+1: